titiler/image/main.py

```diff
@@ -40,6 +41,7 @@
 DEFAULT_STATUS_CODES.update(
     {
         RasterioIOError: status.HTTP_404_NOT_FOUND,
+        RequestValidationError: status.HTTP_400_BAD_REQUEST,
```

Comment:
This is to accommodate with the IIIF specification which will return 400 for all `bad` request (will fastapi will return a 422 when endpoint query parameter are not valid